lib/converters/v2.1.0/converter-v21-to-v1.js

```diff
@@ -16,10 +16,12 @@ _.extend(Builders.prototype, {
      * Converts arrays of v2.1 style auth params to their v1.0.0 equivalent objects.
      *
      * @param {Object} entity - A v2.1 compliant wrapped auth manifest.
+     * @param {?Object} options - The set of options for the current auth cleansing operation.
+     * @param {?Boolean} [options.includeNoauth=false] - When set to true, noauth is set to null.
```

Comment:
Which case would need `noauth` to be removed?

Reply:
At a collection level.